src/plot-data.ts

```diff
@@ -3017,13 +3015,7 @@ export class Interactions {
         plot_data.refresh_axis_coords();
       }
     } else if ((plot_data.rubber_bands[selected_axis_index].length != 0) && !click_on_band && !click_on_border) {
-      // plot_data.rubber_bands[selected_axis_index].reset();
-      plot_data.rubber_bands[selected_axis_index] = new RubberBand(plot_data.rubber_bands[selected_axis_index].attributeName, 0, 0);
```

Comment:
There was already a RubberBand object ? I thought they were just arrays

Reply:
There has never been a rubberband object before I coded it. Here is a previous change I made